test/livebook/remote_intellisense_test.exs

```diff
@@ -89,5 +89,25 @@ defmodule Livebook.RemoteIntellisenseTest do
                insert_text: "hello($0)"
              } in Intellisense.get_completion_items("RemoteModule.hel", context, node)
     end
+
+    test "find modules from apps", %{node: node} do
+      context = eval(do: nil)
+
+      assert %{
+               label: "all_keys/1",
+               kind: :function,
+               detail: ":mnesia.all_keys/1",
+               documentation: """
+               Returns a list of all keys in the table named `Tab`. The semantics of this function is context-sensitive. For more information, see `mnesia:activity/4`. In transaction-context, it acquires a read lock on the entire table.
+
```

Comment:
Don’t match on the documentation, as if Erlang update the docs, our tests break. Perhaps match only on The first sentence. The same for specs, maybe match only on the function name from spec but not the whole signature. :)